commons/c2cgeoportal_commons/models/main.py

```diff
@@ -326,6 +326,9 @@ class TreeGroup(TreeItem):
     __tablename__ = "treegroup"
     __table_args__ = {"schema": _schema}
     __acl__ = [DENY_ALL]
+    __mapper_args__ = {
+        "polymorphic_identity": "treegroup"  # needed for _identity_class
+    }
```

Comment:
We shouldn't need that because it's an "abstract" class...

Reply:
We need that to have treeitem in _identity_class so we get layers in session.identity_map under treeitem key. It is not usefull for the database side, but for the session side. With this I get the tree in 100ms instead of 1s.

Reply:
```
"polymorphic_identity": "abstract"  # For performance issue in mapper (_identity_class)
```